source/server/getting-started/launch-script.rst

```diff
@@ -30,6 +30,13 @@ Windows
 
 Save your Windows launch script as ``launch.bat``.
 
+.. warning::
+   Launching your server in a Command Prompt or PowerShell environment can lead to the server hanging. During this 
+   time, the only messages captured in the log state the server skipped many seconds or minutes worth of ticks. 
+   The cause is the ``Quick Edit Mode`` of these shells. This mode freezes the process when you mark (or highlight) 
+   something for a quick edit (e.g., copy/paste). You can prevent hangs from occurring by not highlighting any text on 
```

Comment:
I would change `hangs` to `this`, mainly because there are lots of other ways a server might hang ;)

Reply:
Changed

Reply:
This seems to have been reverted too.